crypto/rsa_extra/rsa_test.cc

```diff
@@ -967,30 +967,6 @@ TEST(RSATest, CheckKey) {
   ASSERT_TRUE(BN_hex2bn(&rsa->d, kDEuler));
   EXPECT_TRUE(RSA_check_key(rsa.get()));
 
-  // If d is completely out of range but otherwise valid, it is rejected.
```

Comment:
Should we keep this test and validate we always accept keys when d > n?

Reply:
good point, added.